docs/adusting_with_depth_and_pressure.md

```diff
@@ -0,0 +1,155 @@
+(adjustment_section)=
+# Adjusting with Depth and Pressure
+
+The dry_rock model can be adjusted according to depth and pressure:
+
+```yaml
+dry_rock:
+    model:
+        # model type is one of:
+        #  * "polyfit"
+        #  * "friable_sand"
+        #  * "patchy_cement"
+        type: "polyfit"
```

Comment:
Is it idiomatic to use quotes for strings in this library? Seems a bit noisy?

Reply:
Right, I'll do away with those.